crates/ruff_linter/resources/test/fixtures/ruff/RUF101_1.py

```diff
@@ -0,0 +1,13 @@
+"""Regression test for #14531.
+
+RUF101 should trigger here because the TCH rules have been recoded to TC.
+"""
+# ruff: noqa: TCH002
```

Comment:
Would you mind adding an example where it's a non-direct-match redirect. E.g. `TCH` or `TCH0`

Reply:
I tried `# ruff: noqa: TCH`, and it looks like it's not even being detected as a valid code (`ParsedFileExemption::try_extract` returns `Err(MissingCodes)`). Is that what you expected, or am I misunderstanding/broke something? I rolled back to before my changes, and I'm still seeing the `Err(MissingCodes)`, so I'm probably misunderstanding.

Reply:
It's probably me. It's very likely that Ruff doesn't support file-level suppressions for selectors? If that's the case, then we're all good here

Reply:
Yeah I think it has to be an exact code or a catch-all (so `TCH` wouldn't be allowed).